bin/analysis/get_app_analytics.py

```diff
@@ -0,0 +1,120 @@
+import logging
+import emission.core.get_database as edb
+import emission.storage.timeseries.abstract_timeseries as esta
+import pandas as pd
+import matplotlib.pyplot as plt
+import datetime as dt
+import matplotlib.dates as mdates
+
+
+def get_app_analytics():
+    df = pd.DataFrame()
+    for user in edb.get_uuid_db().find():
+        user_df = esta.TimeSeries.get_time_series(user['uuid']).get_data_df("stats/server_api_time", time_query=None)
+        if not user_df.empty:
+            df = df.append(user_df, ignore_index = True)
+
+    df['datetime'] = df.ts.apply(lambda ts: dt.datetime.fromtimestamp(ts))
+    df.ix[df.reading>1, 'reading'] = 1
+    fig, ax = plt.subplots()
+    ax.xaxis.set_major_locator(mdates.WeekdayLocator())
+    ax.xaxis.set_major_formatter(mdates.DateFormatter("%m/%d/%Y"))
+    plt.ylabel('Response time')
+
+    dashboard_df = df[df.name == "POST_/result/metrics/timestamp"]
+    dashboard_df.plot(x="datetime", y="reading", ax=ax, style='+', legend=None)
+    plt.title('Dashboard')
+    fig.savefig('Dashboard.png')
```

Comment:
in a subsequent change, can you please save the figures into a directory (like `app_analytics_results`) so that it is easier to zip them all up and share with you?